src/help/components/entry/Entry.tsx

```diff
@@ -77,15 +76,36 @@ const getCleanTextOptions = (headingLevel: HeadingLevels): IOptions => ({
   },
 });
 
+type HelpEntryContentProps = {
+  data: HelpEntryResponse;
+  upperHeadingLevel?: HeadingLevels;
+};
+
 export const HelpEntryContent = ({
   data,
-  handleClick,
   upperHeadingLevel = 'h1',
-}: {
-  data: HelpEntryResponse;
-  handleClick: MouseEventHandler<HTMLElement>;
-  upperHeadingLevel?: HeadingLevels;
-}) => {
+}: HelpEntryContentProps) => {
+  const history = useHistory();
+
+  // Hijack clicks on content
+  const handleClick = useCallback<MouseEventHandler<HTMLElement>>(
+    (event) => {
+      // Hijack clicks on content but only interested in link clicks
+      if (event.target instanceof HTMLAnchorElement) {
```

Comment:
This might addressed in another PR, but this causes issues with other links in news articles (e.g. to advanced search results, etc...)

Reply:
This logic needs to be handled in the "catch all" logic in the memory router's switch, in a further PR indeed